python/dgl/graphbolt/minibatch.py

```diff
@@ -80,6 +80,18 @@ class MiniBatch:
           or other graph components depending on the specific context.
     """
 
+    query_index: Union[torch.Tensor, Dict[str, torch.Tensor]] = None
+    """
+    Group ids associated with seed nodes / node pairs in the graph, which
+    indicates to which query a seed node / node pair belongs.
+    - If `query_index` is a tensor: It indicates the graph is homogeneous. The
+      value should be corresponding query to given 'seed_nodes' or
+      'node_pairs'.
+    - If `query_index` is a dictionary: It indicates the graph is
+      heterogeneous. The keys should be node or edge type and the value should
```

Comment:
Will the key be the same across different node / edge type?

Reply:
Use `query` or `indexes` or `group_id`